modules/horizon-monitor/README.md

```diff
@@ -0,0 +1,66 @@
+# horizon-monitor
```

Comment:
have more description what this module creates, and also a basic example with required params set and module source pointing to terraform public registry